### DISCUSSION TITLE
«percy/healthcheck» does not return «x-percy-core-version»

### DISCUSSION THREAD

**Discussion by dimacnj** [percy/percy-capybara#178] [Q&A]
Migrated to new CLI https://docs.percy.io/docs/capybara#waiting-for-elements-with-capybara
After running Percy gives a message : `[percy] You may be using @percy/agent which is no longer supported by this SDK. Please uninstall @percy/agent and install @percy/cli instead. https://docs.percy.io/docs/migrating-to-percy-cli`

**Comment by Robdel12** [comment]
Hey @dimacnj! The message there is pretty clear -- you're likely still using `@percy/agent`. You'll have to make sure `@percy/agent` is completely removed and then install `@percy/cli`

**Reply by dimacnj** [reply]
@Robdel12 thx for the answer, but I removed agent from global and from node_modules in my repo, basically just killed all node_modules directory and did `npm i` again

**Reply by dimacnj** [reply]
Proof
npm list @percy/agent                 
/Users/qa
└── (empty)

**Reply by Robdel12** [reply]
Did you upgrade the `percy-capybara` package as well? The only way that message gets printed is if you're still using old versions of the SDKs

**Reply by dimacnj** [reply]
@Robdel12 sure, updated. I checked fetch('percy/healthcheck') and not getting version from 'x-percy-core-version'

what I have
```
===== RESPONSE =====
Code: 200
Headers
        x-powered-by: "Express"
        access-control-allow-origin: "*"
        content-type: "application/json; charset=utf-8"
        content-length: "93"
        etag: "W/\"5d-uvP1Bm5kmxMciT0QWEZg5qRaLic\""
        date: "Tue, 01 Feb 2022 12:00:36 GMT"
        connection: "keep-alive"
        keep-alive: "timeout=5"
Body: {"success":true,"build":{"number":1661,"url":"https://percy.io/***/**/builds/***"}}
```

**Reply by Robdel12** [reply]
> x-powered-by: "Express"

That's still `@percy/agent`. The new CLI doesn't have any dependencies on express

**Reply by dimacnj** [reply]
@Robdel12 I did npm uninstall @percy/agent, what I missed what do you think?

**Reply by Robdel12** [reply]
If you have or had it installed globally, it's likely still hanging around in your path. We highly recommend _not_ installing globally for that reason, it causes a lot of lingering issues. 

`which percy` could help figure out where it's still hanging out at

**Reply by dimacnj** [reply]
I will try to reinstall node to recheck it
